featurex/tests/test_extractors.py

```diff
@@ -95,6 +96,14 @@ def test_saliency_extractor():
         sf = tl.data['SaliencyExtractor'].data['frac_high_saliency']
         assert np.isclose(sf,0.27461971)
 
+def test_tesseract_extractor():
+    pytest.importorskip('pytesseract')
```

Comment:
Is this easily installable from the command line? If so, can you add a line to the .travis.yml file that installs it in the install section? That way it'll get tested.